.github/workflows/portal-chart-test.yaml

```diff
@@ -95,12 +95,12 @@ jobs:
           helm uninstall portal --namespace install
         if: github.event_name != 'pull_request' || steps.list-changed.outputs.changed == 'true'
 
-        # Upgrade the released portal chart version with the locally available chart
-      - name: Run helm upgrade
-        run: |
-          helm repo add bitnami https://charts.bitnami.com/bitnami
-          helm repo add tractusx-dev https://eclipse-tractusx.github.io/charts/dev
-          helm install portal tractusx-dev/portal --version ${{ github.event.inputs.upgrade_from || '2.3.0' }} --set replicaCount=1 --set postgresql.architecture=standalone --wait --wait-for-jobs --namespace upgrade --create-namespace
-          helm dependency update charts/portal
-          helm upgrade portal charts/portal --set replicaCount=1 --set postgresql.architecture=standalone --wait --wait-for-jobs --namespace upgrade
-        if: github.event_name != 'pull_request' || steps.list-changed.outputs.changed == 'true'
+      #   # Upgrade the released portal chart version with the locally available chart
+      # - name: Run helm upgrade
+      #   run: |
+      #     helm repo add bitnami https://charts.bitnami.com/bitnami
+      #     helm repo add tractusx-dev https://eclipse-tractusx.github.io/charts/dev
+      #     helm install portal tractusx-dev/portal --version ${{ github.event.inputs.upgrade_from || '2.3.0' }} --set replicaCount=1 --set postgresql.architecture=standalone --wait --wait-for-jobs --namespace upgrade --create-namespace
+      #     helm dependency update charts/portal
+      #     helm upgrade portal charts/portal --set replicaCount=1 --set postgresql.architecture=standalone --wait --wait-for-jobs --namespace upgrade
+      #   if: github.event_name != 'pull_request' || steps.list-changed.outputs.changed == 'true'
```

Comment:
please uncomment to provide the upgrade check again.
we'd need to add the bitnamilegacy chart in the run